nixos/modules/services/misc/lidarr.nix

```diff
@@ -0,0 +1,47 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.lidarr;
+in
+{
+  options = {
+    services.lidarr = {
+      enable = mkEnableOption "Lidarr";
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.services.lidarr = {
+      description = "Lidarr";
+      after = [ "network.target" ];
+      wantedBy = [ "multi-user.target" ];
+      preStart = ''
+        test -d /var/lib/lidarr/ || {
+          echo "Creating lidarr data directory in /var/lib/lidarr/"
+          mkdir -p /var/lib/lidarr/
+        }
+        chown -R lidarr:lidarr /var/lib/lidarr/
+        chmod 0700 /var/lib/lidarr/
```

Comment:
Alternatively you can use 
```
serviceConfig.StateDirectory=lidarr";
serviceConfig.StateDirectoryMode="0700";
```
see `man systemd.exec`

Reply:
Isn't mode 770 better, so that users in the "lidarr" group also have access?

Reply:
@symphorien Oh, didn't know that. I mostly copied the package and service from Sonarr. Pushed with 0770 as mode as suggeste by @bjornfor